uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -574,13 +576,20 @@ define([
 
         this.supportedFileTypes = _.keys(sharedSettings.fileUploader.allowedMimeTypes);
         role = this.directiveOptions.userRole || 'staff';
+        this.canManage = this.isRole('manager') || this.isRole('admin');
         this._initRequest();
 
         return loadStatuses.call(self)
           .then(function () {
             initOpenMode.call(self);
 
-            return self.isRole('manager') && loadManagees.call(self);
```

Comment:
```js
return this.canManage && loadManagees.call(self);
```
we should keep the number of properties and methods and internal functions to a minimum. Having a `loadManagees()` and a `loadManagee()` functions is confusing and just bloats the component's code. Find an elegant way to handle the exception of having to select only one specific managee in the already existing `loadManagees()` function